styles.css

```diff
@@ -0,0 +1,16 @@
+form {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+input, select, textarea{
+  margin: 10px auto;
+  padding: 7px 0;
+  width: 100%;
+}
```

Comment:
- [ ] [Optional] Good job with this practice! 👏 As the previous code reviewer pointed, the goal of code review is not an opportunity to command or control your student/colleague but to help them improve by lending them your professional eyes to spot what they missed out, and while doing this, it is important to be kind with words, empathetic in conduct and suggestive in recommendations. It will help by letting them know why whatever you suggested that they do was necessary. In your last sentence here, it looks like you forgot to remind him of how important it is to write code according to conventional best practices.